src/git.ts

```diff
@@ -42,9 +42,8 @@ export async function init(): Promise<void> {
     await execute(`git init`, workspace);
     await execute(`git config user.name ${action.name}`, workspace);
     await execute(`git config user.email ${action.email}`, workspace);
-    await execute(`git remote rm origin`, workspace);
-    await execute(`git remote add origin ${repositoryPath}`, workspace);
-    await execute(`git fetch`, workspace);
+    await execute(`git remote add deployment-repo ${repositoryPath}`, workspace);
+    await execute(`git fetch deployment-repo`, workspace);
```

Comment:
I think `deployment-repo` should follow the same naming conventions as the `temporaryDeploymentDirectory` constants.